src/components/Form/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Button = ({
+  icon = "",
+  text = "Click me!",
+  spacingModifierClass = "mr-2 mb-2",
+  linkPath,
```

Comment:
maybe this could equal "/" by default? idk

Reply:
Glad you suggested this. Just tested this, and the app breaks without passing a link. I changed it to default to "/".